doc/notes_on_celery.py

```diff
@@ -60,31 +60,31 @@ def process_revisions(page_id, page_revs):
 @task_postrun.connect()
 def task_postrun(*args, **kwargs):
     # always, no matter exceptions or not, remove database sessions
-    
-    
+    return
```

Comment:
you're right, I will change this.

Best

On Mon, Feb 9, 2015 at 12:00 AM, Aaron Halfaker <notifications@github.com>
wrote:

> In doc/notes_on_celery.py
> <https://github.com/halfak/Difference-Engine/pull/1#discussion_r24301981>:
>
> > @@ -60,31 +60,31 @@ def process_revisions(page_id, page_revs):
> >  @task_postrun.connect()
> >  def task_postrun(*args, **kwargs):
> >      # always, no matter exceptions or not, remove database sessions
> > -
> > -
> > +    return
>
> This is just documentation and it is intended to be interpreted as
> psuedo-code. See "listener = " below.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/halfak/Difference-Engine/pull/1/files#r24301981>.
>



-- 
Amir